auth/setup.go

```diff
@@ -229,18 +229,34 @@ func AddAdminUser(authService Service, user *model.User) (*model.Credential, err
 		return nil, fmt.Errorf("add user to group - %w", err)
 	}
 
-	// Generate and return a key pair
-	creds, err := authService.CreateCredentials(user.Username)
-	if err != nil {
-		return nil, fmt.Errorf("create credentials for %s %w", user.Username, err)
+	var creds *model.Credential
+	if user.AccessKeyID == "" {
+		// Generate and return a key pair
+		creds, err = authService.CreateCredentials(user.Username)
+		if err != nil {
+			return nil, fmt.Errorf("create credentials for %s %w", user.Username, err)
+		}
+	} else {
```

Comment:
Are we ok with zero checks for the access secret key? e.g. we roll with an empty string?